data/hai/unfettered side missions.txt

```diff
@@ -147,6 +147,127 @@ mission "Unfettered: Untrustable"
+			choice
+				`	"That seems like a good deal, I'm in!"`
+					goto deal
+				`	"You can do better than scan that ship, in fact I'm willing to sell it to you."`
+					goto sell
+				`	"My privacy is very important to me. Stop that scan at once."`
+			`	She seems really surprised, but gives a sign to her officers to stop the scanning. "In that case we will erase the information collected. Truly I had no idea you would see it this way. Still, this is such a shame..."`
+				decline
+			label deal
+			action
+				set "unfettered: carrier scans"
+			`	The scan is finished in a record speed, and Mera invites you on her ship to get to the lab. When you arrive on the bridge she is alone. "I like to be able to speak freely, you see," she says as you look at the empty bridge. "You never know who is listening," she says as she sets a course. "I have something to show you. But first, tell me, how did you acquire this Carrier? We Unfettered like to share stories, sometimes around campfires..." she says before clapping her hands. Suddenly the atmosphere of the room changes, and you can see the outside of the ship as if it was no longer there.`
+			choice
+				`	"There is not much to say, I simply bought it."`
+					goto bought
+				`	"I acquired it in a grand battle for freedom!"`
+					goto freedom
+				`	"How did you do that?"`
+					goto explain
+				`	"I prefer not to talk about it."`
+			`	"Suit yourself."`
+				goto testingsite
+			label bought
+			`	"I see you are very humble, captain."`
+				goto testingsite
+			label freedom
+			`	"So you are already Unfettered in your heart!" She sighs. "I hope it means you understand that we do not all desire battle, but we do see it as a necessity."`
+				goto testingsite
+			label explain
+			`	"This?" She says, looking around. "It's just a few projectors, in truth. This was not their primary intended use, however," she says with a chuckle.`
+			label testingsite
+			`	You quickly arrive at a collection of craters, which remind you of the moon. She stops there and takes a deep breath.`
+			`	"I have a new gadget I wanted to show you. If you would like to do the honors, it has never been tested before!"`
+			choice
+				`	(Dewit!)`
+					goto dewit
+				`	"Are you sure it is safe?"`
+			`	"I am pretty confident it will not blow us up, if that is what you are wondering!"`
+			label dewit
+			`	You press the button and the huge turret on top of the ship starts to fire in a thunder of fire and rocks exploding, the power of the shots so big that the ship's thrusters are having a very hard time maintaining the ship in position. You can hear Mera laughing. "It works! It works!" The weapon appears to be pulse based, but the projectiles are much, much bigger, and more unstable as well, it seems.`
```

Comment:
```suggestion
			`	You press the button and the huge turret mounted on the top of the ship starts to fire in a thunder of fire and rocks exploding, the power of the shots so big that the ship's thrusters are having a difficult time maintaining the ship in position. You can hear Mera laughing. "It works! It works!" The weapon appears to be pulse based, but the projectiles are much, much bigger, and more unstable as well, it seems.`
```
> starts to fire in a thunder of fire and rocks exploding

This could use some more work

Maybe something like this:

`begins firing in a stream of bright pulses, exploding the rocks beneath in a hunder of fire`